lib/synapse_pay_rest/http_client.rb

```diff
@@ -7,10 +8,11 @@ class HTTPClient
     attr_accessor :options
     attr_accessor :headers
     attr_accessor :user_id
+    attr_accessor :timeout
 
     def initialize(options, base_url, user_id: nil)
       @options = options
-      user = '|%s' %options['fingerprint']
+      user = '|%s' % options['fingerprint']
```

Comment:
Yes.  On this line, though I was only fixing a warning raised by my code profiler.  

``` 
   `%' after local variable or literal is interpreted as binary operator even though it seems like string literal
```

Now that we have testing tools in place. In a different PR, I would like to go though and fix up some of the stranger code (but I think it is a little out of the scope of this PR).